client/components/fields/name.tsx

```diff
@@ -1,9 +1,13 @@
 import PropTypes from 'prop-types';
+import {getTranslatedValue} from '.';
+import {IFieldsProps} from '../../interfaces';
 
-export const name = ({item}) => item.name || null;
+export const name = ({item, filterLanguage}: IFieldsProps) => getTranslatedValue(filterLanguage, item, 'name') ||
```

Comment:
We're already sending a `language` field when rendering list item fields. (see https://github.com/superdesk/superdesk-planning/blob/develop/client/components/fields/index.tsx#L48). I believe this was done to show CVs items in users language.

These now seem like competing implementations. Rather than add a `filterLanguage` to individual components, it might be better to add a `language` prop to these components.

This could be done by:
1. Add a `language` argument to the `renderFields` function
2. Remove `const language = getUserInterfaceLanguage();` from within the `renderFields` function
3. In `EventItem` / `PlanningItem` components, pass in `filterLanguage || item.language || getUserInterfaceLanguageFromCV()` to the call of `renderFields`
4. Add `language` prop to the list field components where you want to handle this language (such as `name`, `slugline`, `headline` etc)